db/upgrade.php

```diff
@@ -304,5 +304,16 @@ function xmldb_plagiarism_turnitinsim_upgrade($oldversion) {
         upgrade_plugin_savepoint(true, 2020092301, 'plagiarism', 'turnitinsim');
     }
 
+    /**
+     * Remove "/api" from the turnitin URL as its been added to the endpoint constants.
+     */
+    if ($oldversion > 2021011801) {
```

Comment:
@dwinn Can you confirm that this logic is correct for doing the DB change

Reply:
The logic is correct yeah.

However, I would expect the version number here (and below on savepoint) to be today's date, ie 2021020301 with version.php updated to match.

Apart from that good stuff.